qrexec/tools/qrexec_policy_agent.py

```diff
@@ -321,10 +322,23 @@ def can_perform_action(self):
 
 class RPCConfirmationWindow:
     # pylint: disable=too-few-public-methods,too-many-instance-attributes
-    _source_file = pkg_resources.resource_filename(
+    __slots__ = ('_gtk_builder',
```

Comment:
Please avoid using `__slots__`. Generally it isn't pythonic approach, and it exists only when memory usage is critical for large data amounts.